src/configure.py

```diff
@@ -494,10 +495,13 @@ def main(args):
     except Exception as e:
         import errno
 
-        print("Exception during write operation invoked: {}".format(e))
         if e.errno == errno.ENOSPC:
-            print("Not enough space on device!")
-        exit()
+            error_msg = "Exception during write operation invoked: {}. Not enough space on device.".format(
+                e
+            )
+        else:
+            error_msg = "Exception during write operation invoked: {}".format(e)
+        exit_program(ExitCode.NO_SPACE, error_msg)
```

Comment:
This is not the same way as it is done in confiugure.py. You forgot the general error

Reply:
Please explain to me what you mean here? this is the configure.py file? I think this should be a general error, which I'll update that to now, but not sure what you mean by it not being the same as it's done in configure.py?

Reply:
Sorry, I meant service_add.py